src/odemis/driver/andorcam2.py

```diff
@@ -2679,7 +2691,7 @@ def GetTemperatureRange(self, p_mint, p_maxt):
         mint = _deref(p_mint, c_int)
         maxt = _deref(p_maxt, c_int)
         mint.value = -200
-        maxt.value = 50
+        maxt.value = -10
```

Comment:
why did you change this?

Reply:
To be simulate more properly real hardware. Typically, the warmest cooling temperature is below 0... not 50°C. That was the reason that we didn't see with the simulator the bug that this PR fixes.